ethdb/memory_database.go

```diff
@@ -26,23 +26,28 @@ import (
 /*
  * This is a test memory database. Do not use for any production it does not get persisted
  */
+
+// MemDatabase imitates the key-value store levelDB for the test memory database.
```

Comment:
Please delete the top comment altogether ("This is a test memory database ..."). As for the main doc, let's make it a bit nicer (it's not only used for testing for a long time now).

`
// MemDatabase is an in-memory key-value store.
`